ksql-cli/src/test/java/io/confluent/ksql/CliTest.java

```diff
@@ -209,7 +280,12 @@ private static void testCreateStreamAsSelect(String selectQuery, final Schema re
     final String queryString = "CREATE STREAM " + resultKStreamName + " AS " + selectQuery;
 
     /* Start Stream Query */
-    testRunner.test(queryString, build("Stream created and running"));
+    assertRunCommand(
+        queryString,
+        anyOf(
+            equalTo(new TestResult("Stream created and running")),
+            equalTo(new TestResult("Parsing statement")),
+            equalTo(new TestResult("Executing statement"))));
```

Comment:
So now we will be fine receiving any of the above responses?

Reply:
yep - they are all correct for the cli to print out.